contracts/axelarnet-gateway/src/contract/execute.rs

```diff
@@ -230,38 +254,32 @@ fn prepare_msgs_for_execution(
             .change_context(Error::SaveExecutableMessage)?;
     }
 
-    Ok(Response::new().add_events(
+    Ok((
+        vec![],
         msgs.into_iter()
-            .map(|msg| AxelarnetGatewayEvent::Routing { msg }.into()),
+            .map(|msg| AxelarnetGatewayEvent::Routing { msg }.into())
```

Comment:
```suggestion
            .map(|msg| AxelarnetGatewayEvent::Routed { msg }.into())
```

Reply:
saw a [comment](https://github.com/axelarnetwork/axelar-amplifier/blob/main/contracts/axelarnet-gateway/src/events.rs#L12) regarding to the naming, intended to match the [name](https://github.com/axelarnetwork/axelar-amplifier/blob/main/contracts/gateway/src/events.rs#L8) with the other gateway contract

Reply:
damn. terrible.